.travis.yml

```diff
@@ -42,6 +44,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   # Install dependencies for source repos
   - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
+  - export PYTHONPATH=/usr/lib/python2.7/dist-packages:$PYTHONPATH
```

Comment:
This is needed because ``catkin`` depends on ``caktin_pkg``(``python-catkin-pkg``),  which is installed at ``/usr/lib/python2.7/dist-packages`` and Python is installed at ``/opt`` in Travis CI.

Please refer to the log https://travis-ci.org/otamachan/ros_controllers/builds/96009427#L2043